### DISCUSSION TITLE
Any model plots from week 8

### DISCUSSION THREAD

**Discussion by DS4B-ANU** [DS4B-ANU/help#32] [Show and tell]
Show any maps you like from your week 8 workshop, and/or any way(s) you have tried of assessing these models. 

If you're going to show one model - show us what you think is the best model. Also briefly describe the assumptions you made, e.g. background points, predictor variables, model type. Also add things like the AUC score, AIC score, cross validation, any information you have on how you assessed the model. (You can mention also models you rejected, and their scores)

**Comment by Kai-Chi024** [comment]
<img width="749" height="389" alt="Screenshot 2026-05-01 at 11 32 22 am" src="https://github.com/user-attachments/assets/8cb5385e-4909-4433-b39c-da6d5f8b4a34" />

<img width="680" height="284" alt="Screenshot 2026-05-01 at 11 33 04 am" src="https://github.com/user-attachments/assets/96ef21b2-469e-4a26-985f-c40b5e041bd9" />

I use Random background points. Random background is known for its simplicity, though it assumes the entire continent is equally accessible to the species, which is a known limitation. A species with a strictly tropical distribution like the Frilled Lizard likely has a more restricted accessible area.

The GAM was selected as it outperformed the GLM on every metric and in every CV fold. The small drop from training to spatial CV AUC (0.938 -> 0.857) suggests good geographic transferability. The thresholded GAM map correctly recovers the tropical savanna belt across northern Australia consistent with the known IUCN range. GLM was rejected due to overprediction in southern and arid regions.

**Comment by yichenL1** [comment]
<img width="995" height="603" alt="image" src="https://github.com/user-attachments/assets/23413a4e-1c72-495d-a252-2d8ffae90ab6" />

**Bias Correction Background-GAM Model**
Yellow points: occurance
Blue line: Alpha Hull (alpha = 0.8)
Random k-folds AUC: 
0.9186565
0.9221131
0.9165738
0.9198045
0.9175371
Spatial k-folds AUC:
0.9275445
0.9032501
0.9317562
0.9233146
0.9036116

**Comment by NedJohnston** [comment]
# GAM with background points as Accessible Area

<img width="1158" height="821" alt="image" src="https://github.com/user-attachments/assets/2c789e5b-7d10-4c2a-a58c-d3b83807ef1d" />

Variables "Bio15" "Bio18" "Bio03" "Bio05" used.

A buffer size of 280km around the alpha hull was selected in Week 7. 
AUC = 0.797

Random k-fold testing found AUC ranged from 0.76 to 0.79

**Comment by zongyuanzheng** [comment]
<img width="1058" height="664" alt="image" src="https://github.com/user-attachments/assets/3fe12961-c998-467e-939f-3f5bfa21a4b4" />
<img width="311" height="156" alt="image" src="https://github.com/user-attachments/assets/ddf76099-af61-4f13-961c-706c48f7f39a" />
<img width="1070" height="663" alt="image" src="https://github.com/user-attachments/assets/2a60b63a-79ae-491a-a8d0-441fcc426fa1" />
<img width="1096" height="623" alt="image" src="https://github.com/user-attachments/assets/185c762c-72dd-40bb-b9b3-a50b347db6ac" />

1. GAM outperformed GLM across all 5 folds

2. AUC difference (~0.012) is small but consistent: not a fluke of one particular data split

3. Higher explained deviance in GAM (47.2% vs 35.5%):  non-linear species-environment relationships are real and ecologically meaningful

4. Both models achieved AUC > 0.89: excellent discriminative ability overall

5. For conservation decision-making: GAM is recommended for its superior predictive performance

**Comment by aporoshenkova** [comment]
<img width="657" height="565" alt="image" src="https://github.com/user-attachments/assets/b95647e3-e663-4341-89c5-dbdf46c84bcd" />

Model Type: GAM (as GLM predicted too much in southern coastal regions)
Variables: Bio08, Bio09, Bio16, Bio19. (none seemed to be correlated)
Absence points: Environmental Envelope
AUC: 0.889
K-Fold cross validation: 0.872-0.898

**Comment by Xinzhe-Lii** [comment]
<img width="446" height="140" alt="image" src="https://github.com/user-attachments/assets/65a177ba-334b-419c-91db-5552a7ac3570" />
<img width="1270" height="602" alt="image" src="https://github.com/user-attachments/assets/c6e3804e-3c71-4f6a-96f6-652cd082944c" />
From the plot, It seems GAM is better than GLM .

However, when I do the spatial fold to calculate AUC, the result shows GLM is really better than GAM.

So in conclusion, GLM is better when using accessible area background for background data.

**Comment by JingyiZhang111** [comment]
<img width="987" height="527" alt="image" src="https://github.com/user-attachments/assets/570f2fe5-3142-41cf-b067-f911705dad05" />

**Model comparison：**

The GAM consistently outperformed the GLM in both training (AUC: 0.938 vs 0.921) and 5‑fold cross‑validation (mean AUC: 0.937 vs 0.921). The cross‑validation AUCs are almost identical to the training AUCs, indicating no severe overfitting. The small but stable improvement of the GAM suggests that non‑linear responses to some bioclimatic variables better capture the species’ ecological niche. Overall, both models perform well (AUC > 0.92), but the GAM is preferred for final predictions.

**Comment by YifeiSun677** [comment]
Not the best model but something interesting: spatial folds. 
Environmental Envelope Background Model
glm: presence ~ Bio16 + Bio19 + Bio08 + Bio09
gam: presence ~ s(Bio16, k = 4) + s(Bio19, k = 4) + s(Bio08, k = 4) + s(Bio09, k = 4) 
<img width="1323" height="688" alt="61037523-9890-4566-a45d-0d122e0c47de" src="https://github.com/user-attachments/assets/7ce3d633-99b5-44f5-bc65-d93ab6b73e20" />

<img width="1225" height="757" alt="image" src="https://github.com/user-attachments/assets/f52a3257-d650-453f-a18c-b9d1dc841a7d" />
<img width="875" height="380" alt="ca78faee-e10f-4244-a6ce-c1cde1b3ebad" src="https://github.com/user-attachments/assets/1c470aed-0584-4342-87c1-2a67ec79e09c" />
glm seems to be more consistent

**Comment by XiangpuWang** [comment]
<img width="922" height="796" alt="image" src="https://github.com/user-attachments/assets/76d47675-08d4-4dbd-9bf0-dda51eb21824" />

Binomial GAM fitted with mgcv (gam, binomial logit), with smooth terms s(predictor, k = 4) on uncorrelated bioclim layers.

ALA presences cleaned; one sighting per bioclim cell; WorldClim (Australia); 5000 random pseudo-absences (spatSample, set.seed(123))—not real absences, so suitability is relative to that sample. Predictors after |r|>0.7 (findCorrelation): Bio02, Bio08, Bio09, Bio16, Bio19.

Models: GAM preferred (mgcv splines); GLM same covariates, linear logits.

Scores
GLM: AIC 3128.826, D^2≈0.3697, train AUC ≈0.9215, 5-fold mean AUC ≈0.9211.
GAM: AIC 2562.461, dev. explained ≈48.8%, train AUC ≈0.9384, 5-fold mean AUC ≈0.9370.

AIC/deviance/train evaluate() AUC vs CV withheld-fold AUC (random folds; use spatial blocking if extrapolation matters).

**Comment by anu-jackdavies** [comment]
<img width="663" height="398" alt="image" src="https://github.com/user-attachments/assets/8aafe030-6972-4ee5-ad15-51c3032e5315" />
The GAM model had an AUC of 0.89 but this dropped to around 0.8 to 0.75 even when spatial alpha folds were used.

**Comment by LiuLL-01** [comment]
<img width="1470" height="894" alt="image" src="https://github.com/user-attachments/assets/3eef6983-3c46-49bf-8b32-e2fecfb41114" />
I was assigned to use random background points. According to the picture and following AUC calculation, the AUC for GLM is 0.921 and the AUC for GAM is 0.938, so GAM is assumed to be better than GLM.

**Reply by LiuLL-01** [reply]
<img width="1434" height="884" alt="image" src="https://github.com/user-attachments/assets/d6557ecf-724d-430e-8cc7-51d536b7d78f" />
In summary, both models have AUC values above 0.9, which means they both performed well. As the AUC value of GAM is greater than GLM, I perfered GAM.

**Comment by seojin-127** [comment]
1, Background points: Accessible area background
2. Predictor variables: Bio15 (Precipitation Seasonality), Bio18 (Precipitation of Warmest Quarter), Bio03 (Isothermality), Bio05 (Max Temperature of Warmest Month)
3. Model type: GAM
4. Alpha Hull (alpha=5)

<img width="766" height="587" alt="image" src="https://github.com/user-attachments/assets/62b6874f-a2fd-4c01-8a3b-b3f2823c2c22" />


5. AUC score: 0.791
6. AIC score: 4173.26
7. Cross Validation (k=5): random (mean 0.783), spatial (mean 0.648)

**Comment by annabellecaramelle** [comment]
<img width="200" height="129" alt="Screen Shot 2026-05-01 at 12 09 11 pm" src="https://github.com/user-attachments/assets/fa59e343-914e-41c8-98ca-a918e56dc141" />

<img width="618" height="382" alt="Screen Shot 2026-05-01 at 11 57 10 am" src="https://github.com/user-attachments/assets/1836339d-c4e7-40a4-8160-6d2f646d7f58" />


1. GAM mostly outperformed GLM. 
2. I used a bias-corrected background sampling approach.
